net/seafile-server/Makefile

```diff
@@ -33,7 +47,7 @@ define Package/seafile-server
     DEPENDS:=+libarchive +libopenssl +glib2 +libsearpc +seafile-ccnet +seafile-seahub +sqlite3-cli +python-mysql +python-urllib3 \
 		+jansson +libevent2 +libevent2-openssl +zlib +libzdb +libsqlite3 +libmysqlclient \
 		+libpthread +libuuid +bash +procps-ng +procps-ng-pkill +SEAFILE_FUSE_SUPPORT:libfuse $(ICONV_DEPENDS)
-    EXTRA_DEPENDS:=seafile-seahub (=6.3.4-1)
+    EXTRA_DEPENDS:=seafile-seahub (=$(SEAHUB_VERSION_RELEASE))
```

Comment:
can we drop the version specific depends?

I think it's reasonable to assume that "opkg install seafile-server` pulls packages from the same repo, or users themselves are responsible for the versions of server and hub when install .ipk files manually.  With that assumption, we just need to make sure at build time that the two packages are compatible by comparing major.minor versions, leaving out numbers of bugfix and revision.  Less maintain details and more flexibility for the user

Reply:
I agree with that.